.travis.yml

```diff
@@ -224,17 +224,10 @@ jobs:
   - <<: *stage_deploy
     if: tag IS present
     <<: *python_3_7_mixture
-    install: skip
-    script: skip
-    deploy:
-      provider: pypi
-      on:
-        all_branches: true
-      user: jaraco
-      password:
-        secure: RAfz06AINvz7bfij/YhfkAreRqamgxS8a6jSRNxntYhtJke3ZszUbIDag8+n1I+G5XT2LnMhHqPNR7Plc+AeMz7VXTuy+b81Li5kse20NYlPhd7mBVmTUpXtqYQashV5J39F4qkATBLznCOrMEomM07VTXjO/o2hmQuXniab2Uo=
-      distributions: dists
-      skip_cleanup: true
+    install:
+    - python -m pip install tox tox-venv
+    script:
```

Comment:
This won't be necessary too.

Reply:
Done in 31150f6a. I'm slightly reluctant to use an environment variable for a one-off where a command-line switch will do, as the latter provides a nice example for a user as to what to do to release, whereas setting an environment variable for the duration of a command is harder and varies by environments (shell, OS, etc).